service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/ApproveDraftOrdersService.java

```diff
@@ -178,6 +183,9 @@ public Map<String, Object> reviewCMO(CaseData caseData, Element<HearingOrdersBun
                 Element<HearingOrder> reviewedOrder;
 
                 if (!JUDGE_REQUESTED_CHANGES.equals(cmoReviewDecision.getDecision())) {
+                    log.info("Creating CMO_REVIEWED work allocation task for case: {}", caseData.getId());
+                    workAllocationTaskService.createWorkAllocationTask(caseData, WorkAllocationTaskType.CMO_REVIEWED);
+
```

Comment:
not sure this should be in a service - should mostly be in an EventHandler (similar to notifications) if possible